include/arch/arm64/sys_io.h

```diff
@@ -81,6 +81,22 @@ static ALWAYS_INLINE void sys_write32(uint32_t data, mem_addr_t addr)
 	__asm__ volatile("str %w0, [%1]" : : "r" (data), "r" (addr));
 }
 
+static ALWAYS_INLINE uint64_t sys_read64(mem_addr_t addr)
```

Comment:
Not that this really matters, but `addr` should have been `mm_reg_t` as per the prototype definition in `sys/sys_io.h`:
https://github.com/zephyrproject-rtos/zephyr/blob/051e9989bb178c764f940264909ab1611d7a5fe9/include/sys/sys_io.h#L225